.github/workflows/instrumented-tests.sh

```
@@ -0,0 +1,35 @@
+if [ $API_LEVEL -ge 28 ]; then
+    TARGET="google_apis_playstore"
+else
+    TARGET="default"
+fi
+
+PACKAGE="'system-images;android-$API_LEVEL;$TARGET;x86_64'"
+
+set -x
+set +e
+echo "Installing platform tools..."
+$ANDROID_HOME/cmdline-tools/latest/bin/android sdk install "build-tools;37.0.0" platform-tools "platforms;android-$API_LEVEL"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)